### PR TITLE
B-21686: Allow Prime Sim User to update estimated weight and actual weight independently

### PR DESCRIPTION
**Pull request by TevinAdams**
## Summary
Issue was that the Prime Sim User could not update Estimated Weight and Actual Weight Independently. This was happening because there is a default 0 in both text fields so if it is not removed then it would get submitted as 0, which would violate the validation rule of a value of 1 or greater. Prime API already supports updating both fields independently. The issue was the prime sim was submitting the payload with default value of 0.

Is there anything you would like reviewers to give additional scrutiny?

[this article](tbd) explains more about the approach used.

## Verification Steps for the Author

These are to be checked by the author.

- [ ] Tested in the Experimental environment (for changes to containers, app startup, or connection to data stores)
- [ ] Have the Agility acceptance criteria been met for this change?

## Verification Steps for Reviewers

These are to be checked by a reviewer.

- [ ] Has the branch been pulled in and checked out?
- [ ] Have the BL acceptance criteria been met for this change?
- [ ] Was the CircleCI build successful?
- [ ] Has the code been reviewed from a standards and best practices point of view?

### Setup to Run the Code

- [Instructions for starting storybook](https://transcom.github.io/mymove-docs/docs/frontend/setup/storybook)
- [Instructions for starting the MilMove application](https://transcom.github.io/mymove-docs/docs/getting-started/application-setup/)
- [Instructions for running tests](https://transcom.github.io/mymove-docs/docs/getting-started/development/testing)

### How to test

1. Access the Prime Sim
2. Start with a shipment that has no estimated weight nor an actual weight
3. Try updating actual weight with a value greater than 0. Save the changes. Should be successful.
4. To speed up testing so you do not have to create a new shipment null out the actual weight , then update the estimated weight on the shipment as the prime.
5. Try adding a 0 in the field and you will get the validation error.

### Frontend

- [ ] There are no aXe warnings for UI.
- [ ] This works in [Supported Browsers and their phone views](https://transcom.github.io/mymove-docs/docs/adrs/Browser-Support/#minimum-browser-requirements) (Chrome, Firefox, Edge).
- [ ] There are no new console errors in the browser devtools.
- [ ] There are no new console errors in the test output.
- [ ] If this PR adds a new component to Storybook, it ensures the component is fully responsive, OR if it is intentionally not, a wrapping div using the `officeApp` class or custom `min-width` styling is used to hide any states the would not be visible to the user.
- [ ] This change meets the standards for [Section 508 compliance](https://www.ssa.gov/accessibility/andi/help/install.html).

### Backend

- [ ] Code follows the guidelines for [Logging](https://transcom.github.io/mymove-docs/docs/getting-started/development/logging).
- [ ] The requirements listed in [Querying the Database Safely](https://transcom.github.io/mymove-docs/docs/backend/guides/golang-guide#querying-the-database-safely) have been satisfied.

#### Any new migrations/schema changes:

- [ ] Follows our guidelines for [Zero-Downtime Deploys](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#zero-downtime-migrations).
- [ ] Have been communicated to #g-database.
- [ ] Secure migrations have been tested following the instructions in our [docs](https://transcom.github.io/mymove-docs/docs/backend/setup/database-migrations#secure-migrations).

### CODE REVIEW

**Review by danieljordan-caci — CHANGES_REQUESTED** [review]
Able to update the estimated weight independently from the actual weight, and vice versa without error.

Only thing missing is a test to validate in `PrimeUIShipmentUpdate.test.jsx`

**Comment by TevinAdams** [comment]
> Able to update the estimated weight independently from the actual weight, and vice versa without error.
> 
> Only thing missing is a test to validate in `PrimeUIShipmentUpdate.test.jsx`

@danieljordan-caci Tests added.

File: src/pages/PrimeUI/Shipment/PrimeUIShipmentUpdate.test.jsx [hunk 1]
```diff
@@ -452,6 +644,56 @@ describe('successful submission of form', () => {
...
+    updatePrimeMTOShipmentV3.mockReturnValue({});
+
+    render(mockedComponent);
+
+    const actualPickupDateInput = await screen.findByLabelText('Actual pickup');
+    await userEvent.clear(actualPickupDateInput);
+    await userEvent.type(actualPickupDateInput, '20 Oct 2021');
+
+    const actualWeightInput = screen.getByLabelText(/Estimated weight/);
+    await userEvent.type(actualWeightInput, '500');
```
Comment (danieljordan-caci) [inline src/pages/PrimeUI/Shipment/PrimeUIShipmentUpdate.test.jsx:684]: variable names need to be `estimatedWeightInput`

**Review by danieljordan-caci — APPROVED** [review]

**Review by traskowskycaci — APPROVED** [review]
Works! It'll be nice to be able to change these values independently

**Pull request merged**